tests/zstreamtest.c

```diff
@@ -448,14 +448,12 @@ static int basicUnitTests(U32 seed, double compressibility, ZSTD_customMem custo
       if (!ZSTD_isError(r)) goto _output_error;  /* must fail : frame requires > 100 bytes */
       DISPLAYLEVEL(3, "OK (%s)\n", ZSTD_getErrorName(r)); }
 
-    DISPLAYLEVEL(3, "test%3i : dictionary compression with masked dictID : ", testNb++);
-    {   ZSTD_parameters params = ZSTD_getParams(1, CNBufferSize, dictionary.filled);
-        ZSTD_CDict* cdict;
-        params.fParams.noDictIDFlag = 1;
-        params.fParams.contentSizeFlag = 1;  /* test contentSize, should be disabled with initCStream_usingCDict */
-        cdict = ZSTD_createCDict_advanced(dictionary.start, dictionary.filled, 1, params.cParams, customMem);
-        { size_t const initError = ZSTD_initCStream_usingCDict(zc, cdict);
```

Comment:
You mean, the previous test ?
It simply never checked that `dictID` isn't set.
The goal used to be to ensure that `contentSizeFlag=1` wasn't preserved after a call to `ZSTD_initCStream_usingCDict()`.
Obviously, it doesn't make sense anymore.